src/user/wikis/Editor/Editor.scss

```diff
@@ -0,0 +1,50 @@
+.wiki-editor {
+  padding: 50px 50px;
+  .wikis-top-controls {
+    display: flex;
+    margin-bottom: 50px;
+    justify-content: flex-end;
+    .vc {
+      display: flex;
+      align-items: center;
+    }
+    * {
+      margin: 0 5px;
+      display: flex;
+      align-items: center;
+    }
+    .btn {
+      padding-left: 0.25rem !important;
```

Comment:
It's not good practice to use **!important**

Reply:
Done